endpoint.go

```diff
@@ -500,8 +500,10 @@ func (ep *endpoint) sbJoin(sb *sandbox, options ...EndpointOption) error {
 		return err
 	}
 
-	if e := ep.addToCluster(); e != nil {
-		log.Errorf("Could not update state for endpoint %s into cluster: %v", ep.Name(), e)
+	if sb.ingress {
```

Comment:
@mrjana Ok, I will remove specific `ingress` check here. But when the container is going down its probably ok to call only `Leave` without an explicit `DisableService`. In any case we have to be defensive against calls who doesn't explicitly call `DisableService`. That is why in `sbLeave` I didn't remove the `deleteFromCluster` call.